example/privacy/main.go

```diff
@@ -0,0 +1,279 @@
+package main
+
+import (
+	"bytes"
+	"encoding/hex"
+	"flag"
+	"fmt"
+	"math/big"
+	"strconv"
+	"time"
+
+	qlcchain "github.com/qlcchain/qlc-go-sdk"
+	"github.com/qlcchain/qlc-go-sdk/pkg/types"
+	"github.com/qlcchain/qlc-go-sdk/pkg/util"
+)
+
+var flagNodeUrl string
+var flagPriKey string
+
+func main() {
+	flag.StringVar(&flagNodeUrl, "nodeurl", "http://127.0.0.1:19735", "RPC URL of node")
+	flag.StringVar(&flagPriKey, "prikey", "", "private key of account")
+
+	flag.Parse()
+
+	client, err := qlcchain.NewQLCClient(flagNodeUrl)
+	if err != nil || client == nil {
+		fmt.Println(err)
+		return
+	}
+
+	fmt.Println(client.Version())
+
+	demoContractAbi(client)
+
+	//demoMintageContract(client)
+
+	//demoMinerContract(client)
+
+	demoPrivacyDemoKVContract(client)
+}
+
+func demoContractAbi(client *qlcchain.QLCClient) {
+	fmt.Println("============ demoContractAbi ============")
+	rspAddrs := client.Contract.ContractAddressList()
+	fmt.Printf("ContractAddressList:\n%s\n", util.ToIndentString(rspAddrs))
+
+	for _, ca := range rspAddrs {
+		rspAbi, err := client.Contract.GetAbiByContractAddress(ca)
+		if err != nil {
+			fmt.Println("GetAbiByContractAddress", err)
+			return
+		}
+		fmt.Printf("GetAbiByContractAddress:\n%s, %s\n", ca, rspAbi)
+	}
+}
+
+func demoMintageContract(client *qlcchain.QLCClient) {
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
func `demoMintageContract` is unused (unused)